pkg/version/version.go

```diff
@@ -52,12 +65,12 @@ func init() {
 	buildInfo := prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
 			Name: "kubernetes_build_info",
-			Help: "A metric with a constant '1' value labeled by major, minor, git version, git commit and git tree state from which Kubernetes was built.",
+			Help: "A metric with a constant '1' value labeled by major, minor, git version, git commit, git tree state, and Go version from which Kubernetes was built.",
 		},
-		[]string{"major", "minor", "gitVersion", "gitCommit", "gitTreeState"},
+		[]string{"major", "minor", "gitVersion", "gitCommit", "gitTreeState", "goVersion"},
```

Comment:
Shouldn't `Compiler`, `BuildDate` and `Platform` be here also? Or am I missing something...